package.json

```diff
@@ -64,6 +64,7 @@
     "ws": "^7.2.3"
   },
   "devDependencies": {
+    "@textile/threads-core": "^0.1.5-rc.0",
```

Comment:
This sucks I know. Its only a dev dep for now, so hopefully that's still ok? We just need it for the types and for tests.